yargs.js

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/env node
+const yargs = require('yargs')
+const archethic = require('archethic')
+const fs = require('fs')
+const chalk = require('chalk')
+const figlet = require('figlet')
+const fileExtension = require('file-extension')
+
+
+
+yargs.command({
+    command: 'about',
+    describe: 'Welcome',
+   
+
+    handler: function (argv)
+    {   
+        console.log(chalk.green('\n','Hello and Welcome to AeWeb !','\n'))
+        console.log(chalk.blue(figlet.textSync('AeWeb',{font : "Alligator2"})))
+        console.log(chalk.green('\n','Create your Website on top of ArchEthic'))
+        console.log(chalk.green('\n','Version - 1.0.0','\n'))
+        
+    }
+})
+
+yargs.command({
+    
+    command: 'generate-address',
+    describe: 'Generate',
+    builder: {
+        seed: {
+            describe: 'Seed',
+            demandOption: true,  // Required
+            type: 'string'     
+        },
+        index: {
+            describe: 'Index',
+            demandOption: true,  // Required
+            type: 'number'     
+        }
+       
+    },
+    
+    handler: function (argv)
+    {   
+        const address = archethic.deriveAddress(argv.seed, argv.index)
+        console.log(chalk.blue(address))
+        console.log(chalk.green("Add some funds to the generated address !"))
+        console.log(chalk.blue("https://testnet.archethic.net/faucet"))
```

Comment:
This sentence only apply for testnet.
Maybe specify if you are using the testnet go there, otherwise transfer funds from your UCO wallet (in mainnet)